cranelift-simplejit/src/backend.rs

```diff
@@ -221,11 +228,11 @@ impl<'simple_jit_backend> Backend for SimpleJITBackend {
         let size = init.size();
         let storage = if writable {
             self.writable_memory
-                .allocate(size)
+                .allocate(size, align.unwrap_or(0x8))
```

Comment:
It'd be nice to make this a constant, with a comment explaining where it comes from (here it's because it's the alignment of the biggest data type we can write at the moment, `f64`, right?).

Reply:
It took all alignment constants from faerie. https://github.com/m4b/faerie/blob/49aac0ca5b49c6ef95a57a95d8a006ab73e998e5/src/elf.rs#L268-L276